src/System.Management.Automation/utils/StringUtil.cs

```diff
@@ -58,43 +56,42 @@ internal static
         // Typical padding is at most a screen's width, any more than that and we won't bother caching.
         private const int IndentCacheMax = 120;
 
-        private static readonly string[] IndentCache = new string[IndentCacheMax];
+        private static readonly string[] s_indentCache = new string[IndentCacheMax];
 
         internal static string Padding(int countOfSpaces)
         {
             if (countOfSpaces >= IndentCacheMax)
                 return new string(' ', countOfSpaces);
 
-            var result = IndentCache[countOfSpaces];
+            var result = s_indentCache[countOfSpaces];
 
             if (result == null)
             {
-                Interlocked.CompareExchange(ref IndentCache[countOfSpaces], new string(' ', countOfSpaces), null);
-                result = IndentCache[countOfSpaces];
+                Interlocked.CompareExchange(ref s_indentCache[countOfSpaces], new string(' ', countOfSpaces), null);
+                result = s_indentCache[countOfSpaces];
             }
 
             return result;
         }
 
         private const int DashCacheMax = 120;
 
-        private static readonly string[] DashCache = new string[DashCacheMax];
+        private static readonly string[] s_dashCache = new string[DashCacheMax];
 
         internal static string DashPadding(int count)
         {
             if (count >= DashCacheMax)
                 return new string('-', count);
 
-            var result = DashCache[count];
+            var result = s_dashCache[count];
 
             if (result == null)
             {
-                Interlocked.CompareExchange(ref DashCache[count], new string('-', count), null);
-                result = DashCache[count];
+                Interlocked.CompareExchange(ref s_dashCache[count], new string('-', count), null);
+                result = s_dashCache[count];
             }
 
             return result;
         }
     }
 }
-
```

Comment:
Please retain the ending new line.

Reply:
we only need a single blank line at EOF (change removes double blank line)

Reply:
Ah, if that's the case, then we are good :)